build-system-tests/scripts/mega-app-build.sh

```diff
@@ -72,11 +72,6 @@ while [[ $# -gt 0 ]]; do
     shift
 done
 
-# Check if MEGA_APP_NAME is provided
```

Comment:
Name is required so removing this because it's confusing

Reply:
did this just set some default?

Reply:
Yeah